pkgs/by-name/is/isocline/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "isocline";
+  version = "1.0.9";
+
+  src = fetchFromGitHub {
+    owner = "daanx";
+    repo = "isocline";
+    rev = "v${version}";
+    hash = "sha256-9hMvXa9+7XtB2pMQ3mQYccdM0wyscUooMpBwDoh1DiA=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir $out/lib -p
+    mkdir $out/bin -p
+    mkdir $out/include -p
+
+    mv -t $out/lib libisocline.a
+    cp $src/include/*.h $out/include
+
+    ${lib.optionalString (!stdenv.hostPlatform.isMinGW) ''
+      mv -t $out/bin example
+    ''}
+    ${lib.optionalString stdenv.hostPlatform.isMinGW ''
+      mv -t $out/bin example.exe
+    ''}
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "A portable GNU readline alternative";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

> `meta.description` must ... Not start with the definite or an indefinite article


```suggestion
    description = "Portable GNU Readline alternative";
```